### DISCUSSION TITLE
Thoughts on this profile page mockup?

### DISCUSSION THREAD

**Discussion by EnZon3** [EnZon3/Drop#7] [Ideas]
![image](https://github.com/EnZon3/Drop/assets/37979038/2d092190-5588-4533-a1fc-50202ca344f7)
Any feedback would be appreciated!

**Comment by MilesWK** [comment]
I take it the flex boxes are hidden?

**Reply by EnZon3** [reply]
Yeah, they are hidden, of course. I highlighted them just to show the layout.

**Reply by MilesWK** [reply]
In that case I like it very much

**Reply by EnZon3** [reply]
:+1:
